deploy/crds/nmstate.io_nodenetworkstates_crd.yaml

```diff
@@ -61,6 +61,9 @@ spec:
                 of golang struct so we don't need to be in sync with the schema. \n
                 [1] https://github.com/nmstate/nmstate/blob/master/libnmstate/schemas/operational-state.yaml"
               type: object
+            lastSuccessfulUpdate:
```

Comment:
Please use `lastSuccessfulUpdateTime`, it would match the other time attributes.